src/libutil/meson.build

```diff
@@ -0,0 +1,133 @@
+# libutil build file
+#============================================================================
+
+libutil_inc = [include_directories('.')]
+
+
+# dependencies
+#============================================================================
+
+libutil_dep_list = [
```

Comment:
`liblzma_dep`, `zlib_dep`, and `libbz2_dep` could thus be removed.

Reply:
nice catch :)

Reply:
bz2 hasnt been completely removed, its still linked in libstore

Reply:
libutil is still linking against zlib headers, too.

Reply:
i dont think the dependencies has been entirely removed :/

Reply:
After a little local experiment, I think they are not necessary...
Let's see where NixOS/nix#5123 goes!

Reply:
okay :)

Reply:
It's been merged :)

Reply:
cool, removed the build deps